.travis.yml

```diff
@@ -1,9 +1,16 @@
 language: node_js
 node_js:
-  - "10"
+  - "10.4.1"
```

Comment:
Did this fail with just 10 there?

Reply:
Yes, here's the failing PR https://github.com/RetailMeNotSandbox/roux-resolver-webpack-plugin/pull/18

Reply:
Let me include the debug change in here and confirm that build is fine.